index.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
     // - application has ember-cli-fastboot dependency.
     this._needsFastBootSupport = this._config.enabled &&
       this._config.delivery.includes('header') &&
-      new VersionChecker(this.project).for('ember-cli-fastboot').exists();
+      this.project.findAddonByName('ember-cli-fastboot') !== null;
```

Comment:
Ember CLI Version Checker does not work well within yarn workspaces. It reports wrongly that a dependency exists if that dependency is installed in the workspace even if it's not a direct nor indirect dependency of the package.